src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -759,6 +788,16 @@ public function getDate()
     {
         return $this->date;
     }
+
+    public function setInners(array $inners)
```

Comment:
This is out of scope of this PR. Support for such notations should be added in the PropertyInfo component.